<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GS1 Simple Flash Text Generator</string>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Simple Source Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTarget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Usable Flash Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEncodedTarget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoded Usable Flash Text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Generate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>681</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtTarget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtEncodedTarget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtSource</tabstop>
  <tabstop>txtTarget</tabstop>
  <tabstop>txtEncodedTarget</tabstop>
  <tabstop>btnGenerate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
